starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  text-align: justify;
+  font-weight: lighter;
+}
+
+h1 {
+  font-size: 70px;
+  text-align: center;
+  line-height: 600px;
+  letter-spacing: 5px;
+  color: white;
+  height: 600px;
+  background-size: 1000px 600px;
+  background-image: url('/Users/briannacho/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg');
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.text-box {
+  width: 800px;
+  margin-left: 150px;
+  margin-right: 350px;
+}
+
+.title-size {
+  font-size: 25px;
+}
+
+.text-size {
+  font-size: 18px;
+}
+
+.info-pics {
```

Comment:
Ok so it looks like you don't use this class in your html so you won't notice any change.

What you can do, is play with your recipe class, because it's the one that "host" almost all of your content.
And may be create a container with your recipe2 id, try with this code : 

#section2 {
  width: 80%;
  margin: 0 10%;
}

It can be a good start :)